app/javascript/controllers/pdf_controller.js

```diff
@@ -2,15 +2,15 @@ import { Controller } from "@hotwired/stimulus"
 import { jsPDF } from "jspdf"
 
 export default class extends Controller {
-  handleClick(event) {
-    event.preventDefault()
+  handleClick(_event) {
     const doc = new jsPDF("p", "px", "a4")
     const applicationReference = document.getElementById(
       "application-reference",
     ).innerText
     const pdfjs = document.getElementById("site-notice-content")
     doc.html(pdfjs, {
       callback: function (doc) {
+        doc.setFont("Helvetica")
```

Comment:
Do Windows machines have Helvetica? I thought not but my knowledge is probably very out of date.